test/apiv2/10-images.at

```diff
@@ -197,6 +197,11 @@ t POST "build?dockerfile=containerfile" $CONTAINERFILE_TAR 200 \
 t POST libpod/images/prune 200
 t POST libpod/images/prune 200 length=0 []
 
+# compat api must allow loading tar which contain multiple images
+podman pull quay.io/libpod/alpine:latest quay.io/libpod/busybox:latest
+podman save -o ${TMPD}/test.tar quay.io/libpod/alpine:latest quay.io/libpod/busybox:latest
+t POST "images/load" ${TMPD}/test.tar 200 \
```

Comment:
This only checks the 200 status, but does not actually check anything else. Please either
* a) add further checks, or
* b) remove the misleading backslash, and document in comments why you do not actually do any testing

Reply:
@edsantiago thanks for spotting removed '\`

Reply:
I think you maybe missed the part where I requested (a) tests, or (b) an explanation of why you didn't add tests?

Reply:
For example:
```bash
t POST "images/load" ${TMPD}/test.tar 200 \
  .stream="Loaded image: quay.io/libpod/busybox:latest"
t GET libpod/images/quay.io/libpod/alpine:latest/exists  204
```
I for one am confused as to why `.stream` only shows one "Loaded image" message, not two. Maybe because `alpine:latest` was already present? In any case, when doing something like this, I think it's important to:
* confirm the actual results of the POST, not just exit status
* confirm that the image(s) were actually loaded

Reply:
> only shows one "Loaded image" message, not two.
@edsantiago  That's because output was hardcoded to [0] i fixed that in a new commit.

Reply:
Thank you.

I know I can be a jackass about testing. I hope this shows its value.

Reply:
@edsantiago No issues at all. Yes I agree we would have never caught this index hardcoding.